### PR TITLE
Move button examples

### LINKED ISSUE #816: Example: button
Split appropriate existing examples into separate files using functional approach and icache and move into the correct `/src/examples/widgets` subdirectory.

### PR DESCRIPTION
**Pull request by vansimke**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:** Moves the button examples into the new format.

Resolves #816

### CODE REVIEW

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-8t9dww9u0.now.sh

File: src/examples/src/widgets/button/DisabledSubmit.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Button from '@dojo/widgets/button';
+
+const factory = create();
+
+export default factory(function DisabledSubmitButton() {
+	return (
+		<Button type="submit" disabled={true}>
```
Comment (tomdye) [inline src/examples/src/widgets/button/DisabledSubmit.tsx]: this can just be `disabled`, no need for the `={true}`

Comment (vansimke) [reply src/examples/src/widgets/button/DisabledSubmit.tsx]: fixed

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-3u83bpcs5.now.sh

**Comment by dojo-site** [comment]
This pull request has been deployed to:
* dojo.widgets: https://dojowidgets-pme9ceq9h.now.sh

**Pull request merged**